### PR TITLE
add cli helps

### PR DESCRIPTION
**Pull request by JR-USAF**

### CODE REVIEW

File: data_pipeline/work_queue_service_cli.py [hunk 1]
```diff
@@ -127,61 +127,87 @@ def main(periodic_functions, work_queue, period_sec=10):
...
         "--periodic-seconds",
-        default=10
+        default=10,
+        help="Number of seconds to wait before checking if additional jobs can be submitted."
     )
 
     known_args, _ = parser.parse_known_args()
 
     # Dynamically add extra arguments based on worker/queue choice
     if known_args.worker_interface == ARGO_WORKFLOWS_INTERFACE_CLI_CHOICE:
```
Comment (JR-USAF) [inline data_pipeline/work_queue_service_cli.py:180]: I can't get these helps to show up.

**Comment by JN-USAF** [comment]
Pytests pass and the help documentation looks good

**Review by JN-USAF — DISMISSED** [review]

File: data_pipeline/work_queue_service_cli.py [hunk 2]
```diff
@@ -127,63 +162,96 @@ def main(periodic_functions, work_queue, period_sec=10):
...
     )
+    
     parser.add_argument(
         "--with-queue-events",
         type=bool,
-        default=False
+        default=False,
+        help="Flag to signify that logs should be stored on queue state changes. "
+             "The 'event_store_implementation' argument should be set to 'sql-json' "
+             "when including this flag."
```
Comment (AM-USAF) [inline data_pipeline/work_queue_service_cli.py:197]: Is there ever a time when the event_store_implementation would be set to anything other than 'sql-json' when this flag is true? If not, why do we require a user to set it?

File: data_pipeline/work_queue_service_cli.py [hunk 3]
```diff
@@ -127,63 +162,96 @@ def main(periodic_functions, work_queue, period_sec=10):
...
+    parser.add_argument("--namespace", 
+                        help="Kubernetes namespace where ARGO Workflows is running. "
+                            f"Required when worker-interface is set to {ARGO_WORKFLOWS_INTERFACE_CLI_CHOICE}")
+
+    parser.add_argument("--connection-string", 
+                        help="Connection string associated with an external SQL server. "
+                            f"Required when queue-implementation is set to {JSON_SQL_QUEUE_CLI_CHOICE}")
+
+    parser.add_argument("--queue-name", 
+                        help="User defined name queue name. Can be any unique string. "
```
Comment (AM-USAF) [inline data_pipeline/work_queue_service_cli.py:229]: should 'name' be here twice?

File: tests/test_validate_args.py [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            'event_store_implementation': 'none',
+            'with_queue_events': False,
+            'worker_interface_id': 'dummy-id',
+            'endpoint': 'dummy-endpoint',
+            'namespace': 'dummy-namespace',
+            'connection_string': None,
+            'queue_name': None,
+            's3_base_path': 'dummypath',
+            'add_to_queue_event_name': None,
+            'move_queue_event_name': None}
```
Comment (AM-USAF) [inline tests/test_validate_args.py:15]: Do we need to include every flag here? Does it work if we only include the flags required based on the choice?

File: data_pipeline/work_queue_service_cli.py (hunk 2 above)
Comment (JR-USAF) [reply data_pipeline/work_queue_service_cli.py:197]: I added a user suggestion for this. I agree. But it was out of the scope of this ticket

File: tests/test_validate_args.py (hunk 4 above)
Comment (JR-USAF) [reply tests/test_validate_args.py:15]: I figured it is better to be explicit.

File: data_pipeline/work_queue_service_cli.py (hunk 3 above)
Comment (JR-USAF) [reply data_pipeline/work_queue_service_cli.py]: Good catch, fixed.

**Review by CW-USAF — DISMISSED** [review]

**Review by AM-USAF — APPROVED** [review]

**Review by CW-USAF — APPROVED** [review]
I don't know if all the ci/cd checks need to pass, but if not it looks good.

File: tests/test_validate_args.py [hunk 5]
```diff
@@ -0,0 +1,239 @@
+from data_pipeline.work_queue_service_cli import (
+                                    validate_args,
+                                    JSON_S3_QUEUE_CLI_CHOICE, 
+                                    JSON_SQL_QUEUE_CLI_CHOICE, 
+                                    ARGO_WORKFLOWS_INTERFACE_CLI_CHOICE, 
+                                    NO_EVENT_STORE_CLI_CHOICE, 
+                                    SQL_EVENT_STORE_CLI_CHOICE)
+                                                 
+
+def test_validate_args_s3_success():
```
Comment (AM-USAF) [inline tests/test_validate_args.py:10]: I didn't realize pylint/ruff don't catch this: per the docstring standards of our repo, tests should have a one-line docstring at the top of the definition

File: tests/test_validate_args.py [hunk 6]
```diff
@@ -0,0 +1,239 @@
+from data_pipeline.work_queue_service_cli import (
```
Comment (AM-USAF) [inline tests/test_validate_args.py:1]: Top of file should have a docstring

**Review by JN-USAF — APPROVED** [review]

File: tests/test_validate_args.py (hunk 5 above)
Comment (JR-USAF) [reply tests/test_validate_args.py:12]: Done

**Review by AM-USAF — APPROVED** [review]

**Review by BE-USAF — APPROVED** [review]

**Pull request merged**